apps/condo/domains/ticket/schema/TicketAnalyticsReportService.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Generated by `createservice ticket.TicketAnalyticsReportService`
+ */
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/ticket/access/TicketAnalyticsReportService')
+const moment = require('moment')
+const get = require('lodash/get')
+const { createCountersStructure, fetchTicketsForAnalitics, DATE_FORMATS } = require('@condo/domains/ticket/utils/serverSchema/analytics.helper')
+
+
+const TicketAnalyticsReportService = new GQLCustomSchema('TicketAnalyticsReportService', {
+    types: [
+        {
+            access: true,
+            type: 'enum TicketAnalyticsGroupBy { day week month status property }',
+        },
+        {
+            access: true,
+            type: 'input TicketAnalyticsReportInput { where: TicketWhereInput!, groupBy: [TicketAnalyticsGroupBy!] }',
+        },
+        {
+            access: true,
+            type: 'type TicketAnalyticsReportOutput { result: JSON! }',
```

Comment:
please write real type don't use json

Reply:
There is currently no way to determine the result type, because type core does not contain equivalent of `Record<any_filter_string, <Record<any_filter_string, value>>`